go/tools/builders/filter.go

```diff
@@ -37,7 +37,7 @@ type goMetadata struct {
 func readFiles(bctx build.Context, inputs []string) ([]*goMetadata, error) {
 	outputs := []*goMetadata{}
 	for _, input := range inputs {
-		if m, err := readGoMetadata(bctx, input, true); err != nil {
+		if m, err := readGoMetadata(bctx, abs(input), true); err != nil {
```

Comment:
Looks like tests are failing because `//go/builders:filter_test` does not include env.go, which it now needs.

Reply:
Done.